examples/keytab_credentials.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import time
+
+from hazelcast import HazelcastClient
+
+from hzkerberos import TokenProvider
+
+
+def customize_logger(name):
+    lg = logging.getLogger(name)
+    lg.setLevel(logging.DEBUG)
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+    handler.setFormatter(formatter)
+    lg.addHandler(handler)
+
+
+customize_logger("hazelcast")
+customize_logger("hzkerberos")
+
+
+def main():
+    # Create a token provider which uses a principal and its password.
```

Comment:
I think the comments on this and the password credentials sample are mixed. We are mentioning the password in keytab example.